assets/src/components/nav/leftNav.tsx

```diff
@@ -20,11 +18,33 @@ import {
 import inTestGroup, { TestGroups } from "../../userInTestGroup"
 import { togglePickerContainer } from "../../state"
 import NavMenu from "./navMenu"
-import { mapModeForUser } from "../../util/mapMode"
 import Tippy from "@tippyjs/react"
 import { fullStoryEvent } from "../../helpers/fullStory"
 import { OpenView } from "../../state/pagePanelState"
 import { usePanelStateFromStateDispatchContext } from "../../hooks/usePanelState"
+import { LinkData, useNavLinkData } from "./navLinkData"
+
+interface LeftNavLinkProps {
+  linkData: LinkData
+  collapsed: boolean
+}
+
+const LeftNavLink = ({
+  linkData,
+  collapsed,
+}: LeftNavLinkProps): JSX.Element => (
+  <NavLink
+    className={({ isActive }) =>
+      "c-left-nav__link" + (isActive ? " c-left-nav__link--active" : "")
+    }
+    title={linkData.title}
+    to={linkData.path}
+    onClick={linkData.onClick}
+  >
+    <linkData.navIcon className="c-left-nav__icon" />{" "}
```

Comment:
**nitpick:** unnecessary space character (I think this is what caused the snapshot diffs):
```suggestion
    <linkData.navIcon className="c-left-nav__icon" />
```

Reply:
Ooooh I think prettier must have done that without me noticing

Reply:
Yeah, it does that sometimes.